src/Plugin/Thunder/OptionalModule/Liveblog.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\thunder\Plugin\Thunder\OptionalModule;
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * AMP.
+ *
+ * @ThunderOptionalModule(
+ *   id = "thunder_liveblog",
+ *   label = @Translation("Liveblog"),
+ *   description = @Translation("The Liveblog module allows you to distribute blog posts to thousands of users in realtime."),
+ *   type = "module",
+ * )
+ */
+class Liveblog extends AbstractOptionalModule {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+
+    $form = parent::buildForm($form, $form_state);
+
+    $form['thunder_liveblog']['description'] = [
+      '#type' => 'item',
+      '#markup' => $this->t('Register a new account at <a href=":pusher_url" target="_blank">:pusher_url</a>, create a new app and note down your keys. You can provide theme right here or at a later stage on the liveblog settings form',
+        [':pusher_url' => 'http://pusher.com']),
+    ];
+
+    $form['thunder_liveblog']['pusher_app_id'] = [
+      '#type' => 'textfield',
+      '#title' => t('App ID'),
+    ];
+
+    $form['thunder_liveblog']['pusher_key'] = [
+      '#type' => 'textfield',
+      '#title' => t('Key'),
+    ];
+
+    $form['thunder_liveblog']['pusher_secret'] = [
+      '#type' => 'textfield',
+      '#title' => t('Secret'),
+    ];
+
+    $form['thunder_liveblog']['pusher_cluster'] = [
```

Comment:
I filed a ticket https://support.pusher.com/hc/en-us/requests/10412.
When someone opens up a new app, he also gets the cluster as a string, so I think it is ok for now: 
![screenshot 2017-03-08 11 28 17](https://cloud.githubusercontent.com/assets/1248214/23700237/81338082-03f2-11e7-982c-8dacae10ffd6.png)